18F/identity-idp
# LG-6874: Keep doc auth errors available on "Back" for first couple IPP pages

<details>
<summary>GIF Demo</summary>

![back_preserve_error_doc_auth3](https://user-images.githubusercontent.com/90272033/185525166-c8884168-d6d5-4c06-a626-80d8b0727eda.gif)

</details>

### Testing Steps
1. Start doc auth workflow
2. Use an ID that causes an IPP-compatible error (e.g. image resolution)
3. Select the option to verify your identity in-person
4. Select "Back" link or use browser back button
5. The specific error should remain visible
